Source/PropertyTools.Wpf/DataGrid/Definitions/PropertyDefinition.cs

```diff
@@ -299,7 +301,26 @@ public void UpdateDescriptor(Type itemType)
         /// </summary>
         protected void SetEnumItemsSource()
         {
-            this.ItemsSource = Enum.GetValues(this.PropertyType);
+            var values = new List<object>();
+            var enumType = this.PropertyType;
+            if (enumType != null)
+            {
+                var ult = Nullable.GetUnderlyingType(enumType);
+                if (ult != null)
+                {
+                    // TODO: this does not work properly, do we need to use a converter?

```

Comment:
Hi.

My point of view:
1.) The system should behaive the same way regardless with nullable type to deal with.
Means: if we have a int?, double?, enum?, .. the system must do the same things for rendering, editing, ..
Solution to reach this goal:
https://github.com/objorke/PropertyTools/commit/92f51fa6606c541cab9e39ea1a5b3ebd27c1f6c7

2.) To improve the system is all the time a good idea e.g using custom-converter or custom-templates or what else,
but this is beyong the basic system and located in the customizing corner.

regards Dietmar